src/platform/w3dengine/client/w3ddisplay.cpp

```diff
@@ -1397,42 +1397,47 @@ void W3DDisplay::Preload_Texture_Assets(Utf8String texture)
 #define DibSize(lpbi) ((lpbi)->biSize + (lpbi)->biSizeImage + (int)(lpbi)->biClrUsed * sizeof(RGBQUAD))
 
 #define DibPaletteSize(lpbi) (DibNumColors(lpbi) * sizeof(RGBQUAD))
+
 void Create_Bmp_File(const char *filename, const char *data, int width, int height)
 {
 #ifdef BUILD_WITH_D3D8
-    BITMAPINFOHEADER *header = (BITMAPINFOHEADER *)LocalAlloc(LMEM_ZEROINIT, sizeof(BITMAPINFOHEADER));
-    header->biSize = sizeof(BITMAPINFOHEADER);
-    header->biWidth = width;
-    header->biHeight = height;
-    header->biPlanes = 1;
-    header->biBitCount = 24;
-    header->biCompression = 0;
-    header->biSizeImage = 24 * header->biHeight * ((header->biWidth + 7) / 8);
-    header->biClrImportant = 0;
-    HANDLE h = CreateFile(filename, GENERIC_READ | GENERIC_WRITE, 0, nullptr, CREATE_ALWAYS, FILE_ATTRIBUTE_NORMAL, nullptr);
+    // #BUGFIX Simplify code and avoid LocalAlloc. Use the stack.
+    BITMAPINFOHEADER iheader = { 0 };
```

Comment:
I removed `LocalAlloc` alltogether because it is useless to use. Just allocate on stack and it is fine.